starlette/middleware/http.py

```diff
@@ -98,6 +105,3 @@ async def wrapped_send(message: Message) -> None:
 
                 await response(scope, receive, send)
                 return
-
-            if not response_started:
-                raise RuntimeError("No response returned.")
```

Comment:
I don't think we need this here, we can just let the server handle it which is what would happen if we didn't install this middleware. I think we only do it in `BaseHTTPMiddleware` out of necessity / implementation detail.